RocksDBStore/Store.cs

```diff
@@ -0,0 +1,107 @@
+using Neo.Persistence;
+using RocksDbSharp;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Neo.Plugins.Storage
+{
+    internal class Store : IStore
+    {
+        private static readonly byte[] SYS_Version = { 0xf0 };
+        private readonly RocksDb db;
+        private readonly Dictionary<byte, ColumnFamilyHandle> _families = new Dictionary<byte, ColumnFamilyHandle>();
+
+        public Store(string path)
+        {
+            var families = new ColumnFamilies();
+            for (int x = 0; x <= byte.MaxValue; x++)
+                families.Add(new ColumnFamilies.Descriptor(x.ToString(), new ColumnFamilyOptions()));
+            db = RocksDb.Open(Options.Default, path, families);
+
+            ColumnFamilyHandle defaultFamily = db.GetDefaultColumnFamily();
+            byte[] value = db.Get(SYS_Version, defaultFamily, Options.ReadDefault);
+            if (value != null && Version.TryParse(Encoding.ASCII.GetString(value), out Version version) && version >= Version.Parse("3.0.0"))
+                return;
+
+            // Clean all families only if the version are different
+
+            if (value != null)
+            {
+                Parallel.For(0, byte.MaxValue + 1, (x) =>
+                {
+                    db.DropColumnFamily(x.ToString());
+                    db.CreateColumnFamily(new ColumnFamilyOptions(), x.ToString());
```

Comment:
Maybe we don't need this because of:
https://github.com/neo-project/neo-plugins/blob/2c905bd789c33af64f547b2bf8f6ba3a7c2ce2d4/RocksDBStore/Options.cs#L15

Reply:
I tried, but if you remove the families

![image](https://user-images.githubusercontent.com/3167973/70220032-2c80a780-1746-11ea-806c-b782264d5b16.png)


It throw this exception later

```
 Message: 
    Test method neo_plugins.Tests.StoreTest.TestRocksDb threw exception: 
    System.Collections.Generic.KeyNotFoundException: The given key '0' was not present in the dictionary.
  Stack Trace: 
    Dictionary`2.get_Item(TKey key)
    Store.TryGet(Byte table, Byte[] key) line 90
    StoreTest.TestStorage(IStoragePlugin plugin) line 29
    StoreTest.TestRocksDb() line 18
```

Reply:
The exception is thrown by the dictionary, not by RocksDb.